api/app/Console/Commands/SendNotificationsTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\User;
+use App\Notifications\GcNotifyEmailChannel;
+use App\Notifications\Test;
+use Illuminate\Console\Command;
+use Throwable;
+
+class SendNotificationsTest extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'send-notifications:test
+                            {emailAddress : The email address of the user}
+                            {number=1}
+    ';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Send test notifications';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $emailAddress = $this->argument('emailAddress');
+        $numberOfMessages = $this->argument('number');
+
+        $successCount = 0;
+        $failureCount = 0;
+
+        $user = User::where('email', $emailAddress)->sole();
+
+        $notification = new Test('test', GcNotifyEmailChannel::class);
+        for ($i = 0; $i < $numberOfMessages; $i++) {
```

Comment:
One minor thing, I think this is causing one extra email to be sent :confused: I get one additional email than the number I pass in.

Reply:
Sounds like it was just an old queued message.